MCP-Cursor-راهنمای-فارسی.md

```
@@ -0,0 +1,2915 @@
+        raise ExternalServiceError(
+            "خطا در ارتباط با سرویس خارجی",
+            service="external_api",
+            status_code=500
+        )
+    
+    return {"success": True, "data": response}
+```
+
+## توسعه پیشرفته
+
+### 1. استفاده از Async/Await
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)